packages/piro/test/_input_Analysis_ROL_ReducedSpace_TrustRegion_HessianBasedDotProduct_2_parameters.xml

```diff
@@ -0,0 +1,329 @@
+          <Parameter name="Parameters" type="bool" value="0" />
+          <Parameter name="Details" type="bool" value="0" />
+          <Parameter name="Linear Solver Details" type="bool" value="0" />
+          <Parameter name="Stepper Iteration" type="bool" value="1" />
+          <Parameter name="Stepper Details" type="bool" value="1" />
+          <Parameter name="Stepper Parameters" type="bool" value="1" />
+        </ParameterList>
+      </ParameterList>
+      <ParameterList name="Solver Options">
+        <Parameter name="Status Test Check Type" type="string" value="Minimal" />
+      </ParameterList>
+      <ParameterList name="Status Tests">
+        <Parameter name="Test Type" type="string" value="Combo" />
+        <Parameter name="Combo Type" type="string" value="OR" />
+        <Parameter name="Number of Tests" type="int" value="2" />
+        <ParameterList name="Test 0">
+          <Parameter name="Test Type" type="string" value="NormF" />
+          <Parameter name="Tolerance" type="double" value="1.0e-8" />
+        </ParameterList>
+        <ParameterList name="Test 1">
+          <Parameter name="Test Type" type="string" value="MaxIters" />
+          <Parameter name="Maximum Iterations" type="int" value="10" />
+        </ParameterList>
+      </ParameterList>
+    </ParameterList>
+    <ParameterList name="Rythmos">
+      <Parameter name="Num Time Steps" type="int" value="10" />
+      <Parameter name="Final Time" type="double" value="0.1" />
+      <Parameter name="Stepper Type" type="string" value="Explicit RK" />
+      <ParameterList name="Rythmos Stepper">
+        <ParameterList name="VerboseObject">
+          <Parameter name="Verbosity Level" type="string" value="medium" />
+        </ParameterList>
+      </ParameterList>
+      <ParameterList name="Stratimikos">
+      </ParameterList>
+      <ParameterList name="Rythmos Integration Control">
+      </ParameterList>
+      <ParameterList name="Rythmos Integrator">
+        <ParameterList name="VerboseObject">
+          <Parameter name="Verbosity Level" type="string" value="medium" />
+        </ParameterList>
+      </ParameterList>
+    </ParameterList>
+    <ParameterList name="Analysis">
+      <Parameter name="Output Level" type="int" value="2" />
+      <Parameter name="Analysis Package" type="string" value="ROL" />
+      
+      <ParameterList name="Dakota">
+        <Parameter name="Input File" type="string" value="dak.in" />
+        <Parameter name="Output File" type="string" value="dak.out" />
+        <Parameter name="Restart File" type="string" value="dak.res" />
+        <Parameter name="Error File" type="string" value="dak.err" />
+      </ParameterList>
+      <ParameterList name="ROL">
+        <Parameter name="Number Of Parameters" type="int" value="2" />
+        <Parameter name="Response Vector Index" type="int" value="0" />
+        <Parameter name="Parameter Vector Index 0" type="int" value="0" />
```

Comment:
Can you change this so that `index[i] != i`?

Reply:
Done, it works (after changing the order of the expected results).